africanlii/templates/peachjam/home.html

```diff
@@ -36,28 +36,52 @@
       </div>
     </div>
   </section>
-  <section class="pb-5 pt-5">
+  <section class="pb-5 pt-5 bg-light">
     <div class="container">
+      <h1 class="mb-4 heading-underlined text-center">{% trans 'African Union' %}</h1>
       <div class="row">
-        <div class="col-lg-6 mb-4">
-          <h2 class="mb-4 heading-underlined">{% trans 'Regional Bodies' %}</h2>
+        <div class="col-sm-6">
+          <h2 class="mb-4 heading-underlined">{% trans 'African Union Organs' %}</h2>
           <ul class="list-unstyled">
-            {% for author in authors %}
+            {% for au_organ in au_organs %}
               <li>
-                <a href="{% url 'author' author.code %}">{{ author.name }}</a>
+                <a href="{% url 'au_organ_detail_view' au_organ.author.code %}">{{ au_organ }}</a>
               </li>
             {% endfor %}
           </ul>
         </div>
-        <div class="col-lg-6">
+        <div class="col-sm-6">
           <h2 class="mb-4 heading-underlined">{% trans 'Regional Economic Communities' %}</h2>
           <ul class="list-unstyled">
-            {% for locality in localities %}
+            {% for rec in recs %}
+              <li>
+                <a href="{% url 'rec_detail_view' rec.locality.place_code %}">{{ rec }}</a>
+              </li>
+            {% endfor %}
+          </ul>
+        </div>
+        <div class="container">
+          <h2 class="mb-4 heading-underlined">{% trans 'Member States' %}</h2>
+          <a class="mt-3 btn btn-primary"
+             href="{% url 'au_detail_page' %}#member-states">{% trans "Click here to view Member States" %}</a>
+        </div>
+      </div>
+    </div>
+  </section>
+  <section class="pb-5 pt-5">
```

Comment:
This entire section falls away because it is replaced by your new section above.

Reply:
@nickmwangemi this is still outstanding. Delete this entire div.